modules/installation-azure-service-principal.adoc

```diff
@@ -192,6 +192,19 @@ control. For more information, see https://aka.ms/azadsp-cli
 }
 ----
 
+[NOTE]
+====
+If you want to use custom roles with all the required permissions, you must skip the step 6. You can create the service principal for your account by running the following command:
```

Comment:
here the step 6 should be step 4 which create SP with "Contributor" role?

Reply:
@chiragkyal Can you please confirm?

Reply:
I guess this note should be put into the end of [the procedure to create a service principal](https://56274--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_azure/installing-azure-account.html#installation-azure-service-principal_installing-azure-account). If user want to create SP with custom role, should skip step4~step6.

Reply:
I am fine with keeping it either places (after step 4 or step 6) as long as it conveys the message correctly that if users want to use custom role, in place of "Contributor" and "User Access Administrator", then step 4 and step 6 is not required and should be replaced with new command.

Reply:
Addressed - Moved the note in the end of [the procedure to create a service principal](https://56274--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_azure/installing-azure-account.html#installation-azure-service-principal_installing-azure-account)